components/datatypes/commerce.schema.json

```diff
@@ -23,10 +23,6 @@
           "$ref": "https://ns.adobe.com/xdm/data/measure",
           "description": "A product list has been identified as no longer accessible or purchasable by the user."
         },
-        "xdm:checkouts": {
-          "$ref": "https://ns.adobe.com/xdm/data/measure",
-          "description": "An action during a checkout process of a product list, there can be more than one checkout event if there are multiple steps in a checkout process. If there are multiple steps the event time information and referenced page or experience is used to identify the step individual events represent in order."
-        },
         "xdm:productListAdds": {
```

Comment:
Every property needs to have a "title". A lot of properties in this PR are missing one, Please add that.